resolwe/flow/models/entity.py

```diff
@@ -377,6 +377,13 @@ def validate_annotations(self):
         if validation_errors:
             raise ValidationError(validation_errors)
 
+    def save(self, *args, **kwargs):
+        """Propagate the modified time to the collection."""
+        super().save(*args, **kwargs)
+        if self.collection:
+            self.collection.modified = self.modified
```

Comment:
Does `self.modified` contain updated value?

Reply:
This is a good question. I belive it does, at least according to tests.